create_framework/http-foundation.rst

```diff
@@ -121,6 +121,18 @@ To use this component, add it as a dependency of the project:
 
 Running this command will also automatically download the Symfony
 HttpFoundation component and install it under the ``vendor/`` directory.
+A ``composer.json`` and a ``composer.lock`` file will be generated as well:
+
+.. code-block:: json
+
+    {
+        "require": {
+            "symfony/http-foundation": "^2.7"
+    }
+
+``The code block shows the content of the ``composer.json`` file (the actual
```

Comment:
@OskarStark Yeah, but they should only enclose the file name. So you should remove them from the beginning and the end of the paragraph now. :)